frontend/src/components/common/Resource/CreateButton.tsx

```diff
@@ -84,8 +84,9 @@ export default function CreateButton(props: CreateButtonProps) {
 
     const clusterName = getCluster() || '';
 
+    await applyFunc(massagedNewItemDefs, clusterName);
     dispatch(
-      clusterAction(() => applyFunc(massagedNewItemDefs, clusterName), {
+      clusterAction(() => {}, {
```

Comment:
You are waiting for the apply function and then you are calling the cluster action which has the timeout. This does not do what is intended.